source/dagger/gameplay/ping_pong/player_scores.cpp

```diff
@@ -49,7 +49,7 @@ void PlayerScoresSystem::Run()
 
     if (ballOnField == 0)
     {
-        // TODO: set speed to be random in both directions
-        CreatePingPongBall(s_TileSize, ColorRGBA(1, 1, 1, 1), { rand()%10 + 4,rand()%10 + 4,0 },   { 0,rand()%(s_FieldHeight / 2),0 });
+        // TODO: set speed to be random in both directions. Done without if for better performnce 
```

Comment:
remove todo comment pls

use separate variable, so it'll be easier to read the code.

((rand() % 2 + 2)%3-1) - rly interesting way to do that =)